docs/data/run-frontier-squid-container.md

```diff
@@ -0,0 +1,180 @@
+Running Frontier Squid in a Container
+=====================================
+
+Frontier Squid is a distribution of the well-known [squid HTTP caching
+proxy software](http://squid-cache.org) that is optimized for use with
+applications on the Worldwide LHC Computing Grid (WLCG). It has
+[many advantages](https://twiki.cern.ch/twiki/bin/view/Frontier/InstallSquid#Why_use_frontier_squid_instead_o)
+over regular squid for common grid applications, especially Frontier
+and CVMFS. The OSG distribution of frontier-squid is a straight rebuild of the
+upstream frontier-squid package for the convenience of OSG users.
+
+This document outlines how to run Frontier Squid in a Docker container.
+
+## Frontier Squid Is Recommended
+
+OSG recommends that all sites run a caching proxy for HTTP and HTTPS
+to help reduce bandwidth and improve throughput. To that end, Compute
+Element (CE) installations include Frontier Squid automatically. We
+encourage all sites to configure and use this service, as described
+below.
+
+For large sites that expect heavy load on the proxy, it is best to run the proxy on its own host.
+If you are unsure if your site qualifies, we recommend initially running the proxy on your CE host and monitoring its
+bandwidth.
+If the network usage regularly peaks at over one third of the bandwidth capacity, move the proxy to a new host.
+
+
+Before Starting
+---------------
+
+Before starting the installation process, consider the following points (consulting [the Reference section below](#reference) as needed):
+
+1. **Docker:** For the purpose of this guide, the host must have a running docker service
+   and you must have the ability to start containers (i.e., belong to the `docker` Unix group).
+-   **User IDs:** If it does not exist already, the installation will create the `squid` Linux user
```

Comment:
Since it's in a container I don't think we really care about the UID

Reply:
dropped the User IDs line